src/utils/teams.ts

```diff
@@ -78,6 +78,45 @@ export const getTeamHexColor = (color?: DomainColor) => {
   }
 };
 
+export const getTeamHexSecondaryColor = (color?: DomainColor) => {
+  switch (color) {
+    case DomainColor.Aqua:
+      return '--color-teams-yellow-200';
```

Comment:
Seems this is the reason for black color, we don't have yellow 200, only 50 or 100 :
<img width="218" alt="image" src="https://github.com/user-attachments/assets/20d4b264-5a22-4e4d-84b0-0a1091b5ebb2">
<img width="349" alt="image" src="https://github.com/user-attachments/assets/8ba6a5e9-cb59-4f5c-a31b-04085d4d551b">